pkg/builder/builder.go

```diff
@@ -787,18 +822,21 @@ func makeNMAContainer(vdb *vapi.VerticaDB, sc *vapi.Subcluster) corev1.Container
 
 // makeScrutinizeInitContainer builds the spec of the init container that collects
 // scrutinize command
-func makeScrutinizeInitContainer(vscr *v1beta1.VerticaScrutinize) corev1.Container {
-	return corev1.Container{
-		Image: "busybox:latest",
-		Name:  names.ScrutinizeInitContainer,
-		Command: []string{
-			"sh",
-			"-c",
-			"sleep 5",
-		},
-		VolumeMounts: []corev1.VolumeMount{buildScrutinizeVolumeMount(vscr)},
+func makeScrutinizeInitContainer(vscr *v1beta1.VerticaScrutinize, vdb *vapi.VerticaDB, args []string) corev1.Container {
+	cnt := corev1.Container{
+		Image:        vdb.Spec.Image,
+		Name:         names.ScrutinizeInitContainer,
+		Command:      buildScrutinizeCmd(args),
+		VolumeMounts: buildScrutinizeVolumeMounts(vscr, vdb),
 		Resources:    vscr.Spec.Resources,
+		Env:          buildCommonEnvVars(vdb),
 	}
+	cnt.Env = append(cnt.Env, buildNMAEnvVars(vdb)...)
```

Comment:
Reusing `buildNMAEnvVars` is fined for scrutinize, but it might be clear if we also rename the function. This isn't the env vars for the NMA anymore as its used by scrutinize too. How about `buildNMATLSCertsEnvVar`?